src/schemas/block.yaml

```diff
@@ -84,6 +87,12 @@ Block:
           type: array
           items:
             $ref: '#/components/schemas/TransactionSigned'
+    withdrawals:
+      anyOf:
```

Comment:
you shouldn't need `anyOf` here, see `uncles`, it should just be a direct array definition